core/opendaq/security/include/opendaq/permission_manager_private.h

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2022-2023 Blueberry d.o.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+#include <opendaq/permission_manager.h>
+#include <coretypes/dictobject.h>
+
+BEGIN_NAMESPACE_OPENDAQ
+
+/*!
+ * @brief Provides access to private methods of the permission manager.
+ */
+DECLARE_OPENDAQ_INTERFACE(IPermissionManagerPrivate, IBaseObject)
```

Comment:
Is this meant to be accessed my module developer, or is it openDAQ core-only? If the latter is true, we should name this `IPermissionManagerInternal`.